packages/wonder-blocks-form/src/components/__tests__/labeled-text-field.test.js

```diff
@@ -485,3 +486,138 @@ describe("LabeledTextField", () => {
         expect(textField).toHaveProp("autoComplete", autoComplete);
     });
 });
+
+describe("Required LabeledTextField", () => {
+    test("Has * when required prop is true", async () => {
+        // Arrange
```

Comment:
NIT: The style we generally use is to place a newline after `// Arrange` if it is empty, to separate it from `// Act`. I find it tends to help with readability.